sdk/network/Azure.ResourceManager.Network/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.8.1 (2024-04-29)
```

Comment:
The service team is going to release 1.8.0, so just update the changelog and don't release it.

Reply:
OK, that's why no new entry added to the changelog, I will revert the version change.